src/envo/envo-edit.owl

```diff
@@ -41163,6 +41165,24 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105126> "weathering of plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105126> <http://purl.obolibrary.org/obo/ENVO_06105119>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105204> (photosynthetic photon flux density)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Photosynthetically_active_radiation>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105204> "Number of photons in the 400-700 nm range received by a surface for a specified amount of time"@en)
```

Comment:
PATO:density doesn't have a time component. Also, I don't think photons have wavelengths - that's a property of the wave associated with a give (collection of) photons.